CondCore/SiStripPlugins/interface/SiStripPayloadInspectorHelper.h

```diff
@@ -72,58 +73,58 @@ namespace SiStripPI {
   };
 
   /*--------------------------------------------------------------------*/
-  const char * regionType(int index)
+  std::pair<int,const char *> regionType(int index)
   /*--------------------------------------------------------------------*/
   {
     
     auto region = static_cast<std::underlying_type_t<SiStripPI::TrackerRegion> >(index);
 
     switch(region){
-    case SiStripPI::TIB1r: return "TIB L1 r-#varphi";
-    case SiStripPI::TIB1s: return "TIB L1 stereo";
-    case SiStripPI::TIB2r: return "TIB L2 r-#varphi";
-    case SiStripPI::TIB2s: return "TIB L2 stereo";
-    case SiStripPI::TIB3r: return "TIB L3";
-    case SiStripPI::TIB4r: return "TIB L4";
-    case SiStripPI::TOB1r: return "TOB L1 r-#varphi";
-    case SiStripPI::TOB1s: return "TOB L1 stereo";
-    case SiStripPI::TOB2r: return "TOB L2 r-#varphi";
-    case SiStripPI::TOB2s: return "TOB L2 stereo";
-    case SiStripPI::TOB3r: return "TOB L3 r-#varphi";
-    case SiStripPI::TOB4r: return "TOB L4";
-    case SiStripPI::TOB5r: return "TOB L5";
-    case SiStripPI::TOB6r: return "TOB L6";
-    case SiStripPI::TEC1r: return "TEC D1 r-#varphi";
-    case SiStripPI::TEC1s: return "TEC D1 stereo";
-    case SiStripPI::TEC2r: return "TEC D2 r-#varphi";
-    case SiStripPI::TEC2s: return "TEC D2 stereo";
-    case SiStripPI::TEC3r: return "TEC D3 r-#varphi";
-    case SiStripPI::TEC3s: return "TEC D3 stereo";
-    case SiStripPI::TEC4r: return "TEC D4 r-#varphi";
-    case SiStripPI::TEC4s: return "TEC D4 stereo";
-    case SiStripPI::TEC5r: return "TEC D5 r-#varphi";
-    case SiStripPI::TEC5s: return "TEC D5 stereo";
-    case SiStripPI::TEC6r: return "TEC D6 r-#varphi";
-    case SiStripPI::TEC6s: return "TEC D6 stereo";
-    case SiStripPI::TEC7r: return "TEC D7 r-#varphi";
-    case SiStripPI::TEC7s: return "TEC D7 stereo";
-    case SiStripPI::TEC8r: return "TEC D8 r-#varphi";
-    case SiStripPI::TEC8s: return "TEC D8 stereo";
-    case SiStripPI::TEC9r: return "TEC D9 r-#varphi";
-    case SiStripPI::TEC9s: return "TEC D9 stereo";
-    case SiStripPI::TID1r: return "TID D1 r-#varphi";
-    case SiStripPI::TID1s: return "TID D1 stereo";
-    case SiStripPI::TID2r: return "TID D2 r-#varphi";
-    case SiStripPI::TID2s: return "TID D2 stereo";
-    case SiStripPI::TID3r: return "TID D3 r-#varphi"; 
-    case SiStripPI::TID3s: return "TID D3 stereo";
-    case SiStripPI::END_OF_REGIONS : return "undefined";
-    default : return "should never be here";  
+    case SiStripPI::TIB1r: return std::make_pair(1 ,"TIB L1 r-#varphi");
```

Comment:
hi @mmusich - sorry for the delay - how do you plan to use this new index ? (I don't think its used below). The enum has an index already

Reply:
HI @davidlange6, I need it to get the bin number to fill in e.g. https://github.com/cms-sw/cmssw/pull/21265/files#diff-0eaa5baaa2c31b79f1f2bbf7573a8eb3R1454.
I cannot use the indexing already associated to the enum because that one is used to couple it to internal indexing used in `SiStripDetSummary`
https://github.com/cms-sw/cmssw/blob/ee01ed60f0498e0ee4e4ce2fd3f70efe474c96b1/CondFormats/SiStripObjects/interface/SiStripDetSummary.h#L46